Makefile

```diff
@@ -79,11 +81,15 @@ install:
 testbuild: TESTS := $(shell $(GO) list -tags '$(TAGS)' $(PKG))
 testbuild: GOFLAGS += -c
 testbuild:
+	ROOT="$(DIR)"; \
 	for p in $(TESTS); do \
```

Comment:
I don't think so, `go test -c ./...` also doesn't work. That's the reason for this hack here. Better to just lose it.